waspc/src/Wasp/Generator/ServerGenerator/JobGenerator.hs

```diff
@@ -0,0 +1,79 @@
+module Wasp.Generator.ServerGenerator.JobGenerator
+  ( genJobs,
+    genJobFactories,
+  )
+where
+
+import Data.Aeson (object, (.=))
+import Data.Maybe
+  ( fromJust,
+  )
+import qualified GHC.Enum as Enum
+import StrongPath
+  ( Dir,
+    File',
+    Path,
+    Path',
+    Posix,
+    Rel,
+    parseRelFile,
+    reldir,
+    reldirP,
+    relfile,
+    (</>),
+  )
+import Wasp.AppSpec (AppSpec, getJobs)
+import Wasp.AppSpec.Job (Job (perform))
+import Wasp.Generator.ExternalCodeGenerator.Common (GeneratedExternalCodeDir)
+import Wasp.Generator.FileDraft (FileDraft)
+import Wasp.Generator.JsImport (getJsImportDetailsForExtFnImport)
+import Wasp.Generator.Monad (Generator)
+import Wasp.Generator.ServerGenerator.Common
+  ( ServerSrcDir,
+  )
+import qualified Wasp.Generator.ServerGenerator.Common as C
+
+-- | TODO: Make this not hardcoded!
+relPosixPathFromJobFileToExtSrcDir :: Path Posix (Rel (Dir ServerSrcDir)) (Dir GeneratedExternalCodeDir)
+relPosixPathFromJobFileToExtSrcDir = [reldirP|../ext-src|]
+
+data JobFactory = PassthroughJobFactory
+  deriving (Show, Eq, Ord, Enum, Enum.Bounded)
```

Comment:
Hmm, when I remove `deriving Enum` it says I cannot use `enumFrom` due to missing instance. Same with `Enum.Bounded` when using `minBound`. Perhaps I misunderstood the note, however?

Reply:
No no, it was a misunderstanding.

You don't need to import `GHC.Enum` to use the `Bounded` type class, it's available by default. Instead of writing `  deriving (Show, Eq, Ord, Enum, Enum.Bounded`, you can just write ` deriving (Show, Eq, Ord, Enum, Bounded)` and remove the import `import qualified GHC.Enum as Enum`.

Reply:
Btw I don't think we need Enum any more? Or Bounded?

Reply:
We just looked at it, it's still needed (Martin confirmed) :)

Reply:
Yeah definitely required to derive those two, but we can change `Bounded` as you said and remove that Prelude import as well. thanks👍🏻